test/jdk/jdk/incubator/vector/Byte128VectorTests.java

```diff
@@ -5454,6 +5454,55 @@ static void REVERSEMaskedByte128VectorTests(IntFunction<byte[]> fa,
     }
 
 
+
+
+
+
+
+
+
```

Comment:
Do you have plan to refine the template to remove this block of space ? Perhaps there have some other style issues in the template. I suppose it would be better to address them before upstreaming it to JDK main line.

Reply:
As suggested by @XiaohongGong we added different template generation commands in top level shell script one for each valid primitive type. These extra spaces is an artifact of that.